### PR TITLE
Implement Calendar Scheduling functionality between multiple people.

### PR DESCRIPTION
**Pull request by Yojairo**

### CODE REVIEW

File: portfolio/src/main/webapp/scripts/index.js [hunk 1]
```diff
@@ -12,16 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+google.charts.load("current", { packages: ["corechart"] });
+google.charts.setOnLoadCallback(drawChart);
+
+function drawChart() {
```
Comment (allyons-google) [inline portfolio/src/main/webapp/scripts/index.js:18]: Is this related to the same pull request?

Comment (Yojairo) [reply portfolio/src/main/webapp/scripts/index.js:18]: I believe it's part of the previous pull request, which this one is based on
